Misc/Positions_Area Positions 3 UK.txt

```diff
@@ -41,10 +44,10 @@ London Control S22:London Control:127.820:LWS:L:LON:CTR:-:-:0301:0377:N051.18.21
 London Control S16:London Control:136.600:L16:L:LON:CTR:-:-:0301:0377::
 London Control S17:London Control:128.420:L:L:LON:CTR:-:-:0301:0377:N051.32.26.870:W002.43.29.830
 ;TC Midlands
-London Control:London Control:130.920:TCW:L:LTC:CTR:-:-::
+LTC_M_CTR:London Control:121.020:TCM:L:LTC:CTR:-:-:0301:0377:N052.07.30.663:W000.03.23.581
```

Comment:
Might be a good idea to remind mentors to only use LON_M_CTR (or variants of) for mentoring in future, even if mentoring on a TC position. LTC_M_CTR would be confusing...